about.md

```diff
@@ -6,9 +6,11 @@ title: About
 We meet monthly and quarterly to discuss and learn about the Linux family of operating systems and other free-and-open source software.
 
 ## History
+###Fort Wayne LUG 0.0: Genesis
+In 1997, an as-yet-unnamed figure started a Linux mailing list and IRC channel for the Fort Wayne area. A person claiming to be this figure recently responded to one of our user surveys Is this person named Joe? Satoshi Nakamoto? Keyser Söze? Kilroy? No one currently involved with the group seems to know, and if the person in question has ever fully identified him or herself in the past, that knowledge is now lost. All we have is a still semi- anonymous survey response, and some ancient (in Internet time) writings: http://fwlug.tripod.com/fwlug.html. According to the ancient Tripod tablets, two persons named Joe began separate groups around this time. Until such time as the person(s) responsible step(s) forward, the precise origins and timeline of the 1990s Fort Wayne LUG will remain a mystery.
```

Comment:
Second sentence is missing its period.

"semi-anonymous" should not have a space after the hyphen.

Should use formal Markdown hyperlink syntax for the Tripod URL rather than rely on the markdown interpreter parsing the URL and doing it for you.